src/compiler/types.ts

```diff
@@ -5860,6 +5860,7 @@ namespace ts {
         /** Determines whether we import `foo/index.ts` as "foo", "foo/index", or "foo/index.js" */
         readonly importModuleSpecifierEnding?: "minimal" | "index" | "js";
         readonly allowTextChangesInNewFiles?: boolean;
+        readonly usePrefixAndSuffixTextForRename?: boolean;

```

Comment:
Based on my discussion with you, it sounds like this is less a *user* preference so much as a per-editor preference. But some of these other options seem like that too, so perhaps this is the right place.

Reply:
I understood this was the intended place for editor opt-in settings.  It's possible that some will be exposed in the settings UI in the future.

Reply:
Just think of the editor as a "user" of the server. :wink: